src/Home.js

```diff
@@ -2,10 +2,16 @@ import React from "react";
 import { Link } from "react-router-dom";
 import { SlUser, SlLogout, SlHome } from "react-icons/sl";
 import { MdLocalPhone, MdVideocam, MdMenu } from "react-icons/md";
+import isJwtExpired from "./JwtUtil";
 
 class Home extends React.Component {
   render() {
-    if (this.props.statusCode !== 200) {
+    /*
+  Remember to add a header X-REQUEST-USER which contains email of current user to authenticate JWT token for future api calls
+  currently unauthenticated endpoints are ["/health", "/signin", "/signup", "/encrptionKey"]
+  all other endpoints would require 2 headers in future - Authorization (jwt token) and X-REQUEST-USER (current user email)
+  */
+    if (this.props.statusCode === 200 || !isJwtExpired()) {
```

Comment:
```suggestion
    if (this.props.statusCode === 200 && !isJwtExpired()) {
```

Reply:
can we do this? will isJwtExpired become False as soon as we login?

Reply:
anyways its temporary, after adding backend call for getting chat history we can verify jwt with backend and render home page accordingly

Reply:
jwt also need to be improved with session id

Reply:
> can we do this? will isJwtExpired become False as soon as we login before redirecting to home?

after login it will try to render private route which checks if jwt is set in sessionStorage and checks if it is expired

Reply:
> > can we do this? will isJwtExpired become False as soon as we login before redirecting to home?
> 
> after login it will try to render private route which checks if jwt is set in sessionStorage and checks if it is expired

If we do this later we can avoid this Line entirely and make pvtroute responsible for all routings

Reply:
okay we can add this statusCode check in private route and render elements independently